app/models/video.rb

```diff
@@ -1,14 +1,24 @@
 class Video < ActiveRecord::Base
   include Sluggable
+
   belongs_to :category
+  has_many :reviews, -> { order('created_at DESC') }
 
   validates :title, presence: true, uniqueness: true
-  validates :tagline, presence: true, length: {minimum: 10}
+  validates :tagline, presence: true, length: { minimum: 10  }
 
   sluggable_column :title
 
   def self.search_by_title(search)
     return [] if search.blank?
-    where("lower(title) LIKE ?", "%#{search.downcase}%").order('created_at DESC')
+    where("lower(title) LIKE ?", "%#{search.downcase}%").order 'created_at DESC'
```

Comment:
Actually I just realized you are using both `lower` and `downcase`. You don't need to do both. If you use `ILIKE`, you don't need to do either of those things.